packages/block-library/src/cover-image/index.js

```diff
@@ -152,7 +152,7 @@ export const settings = {
 
 			const classes = classnames(
 				className,
-				contentAlign !== 'center' && `has-${ contentAlign }-content`,
+				contentAlign !== 'undefined' && `has-${ contentAlign }-content`,
```

Comment:
I haven't tested the code, but this looks like it should probably be `undefined`, not `'undefined'`.

Reply:
I think we should use the real undefined instead of string here. I think the problem of receiving undefined strings happens because we have a default value of center on contentAlign attribute. I guess if we remove the default the problem gets solved.